docs/content/deployment/deploying_aiflow_server.md

```diff
@@ -1 +1,155 @@
 # Deploying AIFlow Server
+
+In this guide, we demonstrate how to deploy an AIFlow server.
+
+```{note}
+If you use Airflow as the scheduler, the AIFlow server should run on the same machine with the Airflow
+Scheduler, as they rely on the filesystem to submit workflows.
+```
+
+## Starting an AIFlow server
+
+In this section, we will show you how to use the default configuration to start an AIFlow server.
+
+### Generate default configuration
+
+To generate the default configuration of AIFlow server, you can run the following command:
+
+```bash
+init-aiflow-env.sh
+```
+
+This command will generate the [default configuration](default_aiflow_server_config) file `aiflow_server.yaml` in
+the `$AIFLOW_HOME` directory or `$HOME/aiflow` directory if `$AIFLOW_HOME` is not set.
+
+```{note}
+If the config file already exist, the script will not overwrite the config. If you intend to overwrite 
+your existing config, you need to remove it manually and then run the script again.
+```
+
+You can refer to [here](configuration) if you want to learn all the configuration you can change.
+
+### Start the AIFlow server
+
+```{note}
+AIFlow server requires notification server to work. Please make sure you have 
+[deployed a notification server](./notification_server.md) and configure the notification uri in the AIFlow server 
+config file accordingly. 
+```
+
+You can start the AIFlow server with the following command.
+
+```bash
+start-aiflow.sh
+```
+
+It will start the AIFlow server and AIFlow web server in background processes. You can check the log at
+`$AIFLOW_HOME/logs` or `$HOME/aiflow/logs` directory. The pid of the background process is stored at 
```

Comment:
It's recommended to tell the use which the log file is of the AIFlow server or webserver.